README.md

```diff
@@ -31,36 +41,21 @@ data "aws_availability_zones" "available" {
     - [d/aws_route_table: 'destination_prefix_list_id' attribute set for managed prefix list destinations #17347](https://github.com/hashicorp/terraform-provider-aws/pull/17347)
  Open)
 
-- The following [lab](https://www.eventbox.dev/published/lesson/wavelength-v2/prerequisites.html) covers the intent of the MVP. We want to achieve:
-  - One Region with Multiple region-based and wavelength availability zones.
-  - Needed supporting VPC infrastructure (optional)
-  - AKS cluster with
-    - control plane in local az
-    - worker nodes across local and wavelength az
-  - Consul
-
 ## Warnings
 
-- [wl_route_tables.tf](wl_route_tables.tf) is commented out until the issues above are fixed.
+- [wl_route_tables.tf](wl_route_tables.tf) is commented out until the issues above are fixed. You will need to create these manual via scripts or the console.
 
 ## Module
 
 ```hcl
-module "wavelength_zone" {
-  source            = "./modules/vpc"
-  create_vpc        = false
-  availability_zone = "us-east-1a"
+module "wavelength" {
+  source     = "github.com/jsosulska/tf-wavelength"
```

Comment:
Consider renaming this module for future usage to follow the upstream [guidelines](https://www.terraform.io/docs/registry/modules/publish.html)

For example: `terraform-aws-wavelength-zone` or `terraform-aws-wavelength-deployment`